pinot-query-runtime/src/test/resources/queries/Aggregates.json

```diff
@@ -20,11 +20,6 @@
       }
     },
     "queries": [
-      {
```

Comment:
Query on H2 database is returning an INT. Couldn't find a better workaround other than removing this test for H2.

Reply:
Can we comment out this test for now instead of removing ?

Will `SELECT AVG(CAST int_col AS DOUBLE) FROM FOO` work ?